### PR TITLE
fix-api-alt-names

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/giantswarm/issues/3560

### CODE REVIEW

File: pkg/v5/key/key.go [hunk 1]
```diff
@@ -11,6 +11,11 @@ import (
 	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// return alt-names for api certs
```
Comment (JosephSalisbury) [inline pkg/v5/key/key.go]: Godoc is funny here. Should be

`APIAltNames returns the alt names for API certs.`

Comment (rossf7) [reply pkg/v5/key/key.go]: Yes also APIAltNames should take in clusterGuestConfig as a param. You can call ClusterID in APIAltNames.

Comment (calvix) [reply pkg/v5/key/key.go]: @rossf7 i followed similar key function that is also used in this scope  `CertConfigName` which also takes only string

Comment (rossf7) [reply pkg/v5/key/key.go]: OK I'd say that one was wrong too but its not a blocker.

**Comment by calvix** [comment]
quick question this fixes monitoring for azure and maybe all other newly created clusters, should i include this also in the older version or only in the latest?

**Comment by rossf7** [comment]
Since this is a small change and a bug fix I think also fixing in v4 is fine.

**Pull request merged**